implementation/scripts/circleci-provision-external-contracts.sh

```diff
@@ -12,7 +12,7 @@ function fetch_bonded_keep_vendor_address() {
 }
 
 function set_bonded_keep_vendor_address() {
-  sed -i -e "/BondedECDSAKeepVendorAddress/s/0x[a-fA-F0-9]\{0,40\}/${BONDED_ECDSA_KEEP_VENDOR_ADDRESS}/" ./implementation/migrations/externals.js
+  sed -i -e "/BondedECDSAKeepVendorAddress/s/0x[a-zA-Z0-9]\{0,40\}/${BONDED_ECDSA_KEEP_VENDOR_ADDRESS}/" ./implementation/migrations/externals.js
```

Comment:
Do you think it's worth dropping a comment here why it's `a-zA-Z` instead of `a-fA-F`? Looking at this PR it's obvious, but I am not sure if this will be obvious for future-us looking just at this file.

Reply:
I would treat it as just a temporary solution. We should keep addresses in a format safer to parse and replace (e.g. `json`), so we can change `sed` and regexp to e.g. `jq` we use in this script anyways.

Reply:
Then maybe a TODO?